### DISCUSSION TITLE
Comment column can have any name the user chooses, there could be multiple comment columns per metric

### DISCUSSION THREAD

**Discussion by emailic** [langfuse/langfuse#1500] [Ideas]
### Describe the feature or potential improvement

comment can be seen  in the score table under trace detail,  which is great.
However, I believe it would also be useful to have that information visible on the _Datasets >dataset_name> run_name>Dataset Run>Dataset Item_ page, _Runs_ subsection. It would also be useful if more than one comment could be added, and if we had control over how those comment columns are called in each run.

**Comment by marcklingen** [comment]
Currently the comment field (string) is rendered on each score

![CleanShot 2024-03-26 at 10 44 49](https://github.com/langfuse/langfuse/assets/2834609/a85a49cd-5acb-4c04-82dc-82483596cb9c)

do you suggest rendering the comments directly in this table?
